security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -8,24 +8,48 @@ toc::[]
 
 == Purpose
 
-Machine Config Operator certificates are used to secure connections between the Red Hat Enterprise Linux CoreOS (RHCOS) nodes and the Machine Config Server.
+This certificate authority is used to secure connections from nodes to Machine Config Server (MCS) during initial provisioning.
+
+There are two certificates:
+. A self-signed CA, the MCS CA
+. A derived certificate, the MCS cert
+
+=== Provisioning details
+
+{product-title} installations using {op-system-first} are installed using Ignition. This process is split into two parts:
+
+. An Ignition config is created which is references a URL for the full configuration served by the MCS.
+. The Ignition config manifests as a `worker.ign` file created by the `openshift-install` command for user-provisioned infrastucture installation methods. For installer-provisioned infrastructure installation methods that use the Machine API Operator, this configuration appears as the `worker-user-data` secret.
 
 include::snippets/mcs-endpoint-limitation.adoc[]
 
 .Additional resources
 
+* xref:../../post_installation_configuration/machine-configuration-tasks.adoc#understanding-the-machine-config-operator[Understanding the Machine Config Operator].
+
 * xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin].
 
+=== Provisioning chain of trust
+
+The MCS CA is injected into the Ignition configuration under the `security.tls.certificateAuthorities` configuration field. The MCS then provides the complete configuration with a webserver using the MCS cert.
```

Comment:
In the second sentence, the phrase "with a webserver using the MCS cert" is unclear. Does it mean that the MCS provides the complete configuration by using the webserver and the MCE cert?

Reply:
I think it's "using the MCS cert that is hosted on the web server", but i'm not totally sure. @cgwalters can you weigh in here?